docs/src/pages/guides/composition.md

```diff
@@ -23,3 +23,66 @@ WrappedIcon.muiName = 'Icon';
 ```
 
 {{"demo": "pages/guides/Composition.js"}}
+
+## Component property
+
+Material-UI allows you to change the root node that will be used for a component via a property called `component`.
```

Comment:
"that will be rendered"?

Reply:
"via a property called `component`." - this feels slightly redundant given the example that follows, but I don't have a strong opinion.